eng/versioning/version_client.txt

```diff
@@ -124,6 +124,11 @@ com.microsoft.azure:spring-cloud-azure-appconfiguration-config;1.3.0;1.4.0-beta.
 com.microsoft.azure:spring-cloud-azure-feature-management-web;1.3.0;1.4.0-beta.1
 com.microsoft.azure:spring-cloud-azure-feature-management;1.3.0;1.4.0-beta.1
 com.microsoft.azure:spring-cloud-starter-azure-appconfiguration-config;1.3.0;1.4.0-beta.1
+com.azure.spring:azure-spring-cloud-appconfiguration-config-web;2.0.0-beta.1
```

Comment:
Formatting is slightly off here, it is missing the current version.

`<artifact identifier>;<last release GA, or beta if no GA>;<current development version>`

So, this should be:

```suggestion
com.azure.spring:azure-spring-cloud-appconfiguration-config-web;2.0.0-beta.1;2.0.0-beta.1
```